Dockerfile

```diff
@@ -1,19 +1,12 @@
-# Build using the minimum supported Golang version (match go.mod)
-FROM golang:1.18 as builder
-
-WORKDIR /build
-
-COPY . .
-RUN go get -v -t -d ./...
-RUN make build
-
-FROM scratch
-WORKDIR /opt/fortigate_exporter
-
-COPY --from=builder /build/target/fortigate-exporter .
-COPY --from=builder /etc/ssl/certs/ca-certificates.crt .
-ENV SSL_CERT_DIR=/opt/fortigate_exporter
-
-EXPOSE 9710
-ENTRYPOINT ["./fortigate-exporter"]
-CMD ["-auth-file", "/config/fortigate-key.yaml"]
+ARG ARCH="amd64"
+ARG OS="linux"
+FROM quay.io/prometheus/busybox-${OS}-${ARCH}:glibc
+LABEL maintainer="The Prometheus Authors <prometheus-developers@googlegroups.com>"
+
+ARG ARCH="amd64"
+ARG OS="linux"
+COPY .build/${OS}-${ARCH}/fortigate_exporter /bin/fortigate_exporter
```

Comment:
.build seems not used anywhere during the build - hence make is failing on the docker image (at least for me)

Reply:
This is standard Prometheus build templating. All of our CI pipelines run `promu crossbuild` to produce the `.build` artifacts for tar.gz and Docker output.